worker/src/main/java/gov/cms/ab2d/worker/processor/PatientClaimsProcessorImpl.java

```diff
@@ -41,50 +41,50 @@
     private final BFDClient bfdClient;
     private final FhirContext fhirContext;
 
-    @Trace(async = true)
-    @Async("patientProcessorThreadPool")
     /**
      * Process the retrieval of patient explanation of benefit objects and write them
      * to a file using the writer
      */
-    public Future<Void> process(PatientDTO patientDTO, final StreamHelper helper, OffsetDateTime attTime,
```

Comment:
Is the purpose of this to just reduce the # of method parameters?

Reply:
yes